package.json

```diff
@@ -8,7 +8,9 @@
     "dev": "nodemon -w src --config --ext js,graphql --exec \"babel-node src\"",
     "build": "npm run cleanup && babel src -D -d dist",
     "start": "node dist/index.js",
-    "cleanup": "rm -rf dist"
+    "cleanup": "rm -rf dist",
+    "db-migrate": "npx sequelize db:migrate",
+    "db-reset": "npx sequelize db:migrate:undo:all"
```

Comment:
I don't think we need this script. I'd keep as few scripts as possible in the package.json file. Any script can be run using npx anyway...

Reply:
do you mean both scripts?

Reply:
Yes!